docs/README.md

```diff
@@ -55,11 +55,14 @@ If you'd like to run the agent manually rather than as a service, you can do tha
 stanza
 
 # Supported flags:
---config      The location of the agent config file (default: ./config.yaml)
---plugin_dir  The location of the plugins directory (default: ./plugins)
---database    The location of the offsets database file. If this is not specified, offsets will not be maintained across agent restarts
---log_file    The location of the agent log file. If not specified, stanza will log to `stderr`
---debug       Enables debug logging
+--config          The location of the agent config file (default: ./config.yaml)
+--plugin_dir      The location of the plugins directory (default: ./plugins)
+--database        The location of the offsets database file. If this is not specified, offsets will not be maintained across agent restarts
+--log_level       The log level of the agent logger (default: INFO)
+--log_file        The location of the agent log file. If not specified, stanza will log to `stdout`
+--max_log_size    The maximum size of the agent log file in MB before rotating (default: 10)
+--max_log_backups The maximum number of agent log files to retain when rotating (default: 5)
+--max_log_age     The maximum number of days to retain a rotated agent log file (default: 7)
```

Comment:
I agree w/ @jsirianni's point about maintaining backwards compatibility for v1.x.

For v2, we should consider moving all of these into the config file.

Reply:
I had the same thought about the config file, since we're ending up with quite a few flags now. I decided against it because it was such a substantial change, but I agree we could probably fit it in for v2.

Reply:
Can we write a ticket for this in pivotal so it's captured?

Reply:
Captured this feature request [here](https://www.pivotaltracker.com/story/show/180329915)